docs/application/native/guides/alarm/alarms.md

```diff
@@ -139,15 +139,19 @@ ret = alarm_schedule_with_recurrence_week_flag(app_control, &date,
 ```
 
 <a name="scenario_4"></a>
-## Listing All Scheduled Alarms and Canceling an Alarm
+## Managing Scheduled Alarms
 
 To list all scheduled alarms, use the `alarm_foreach_registered_alarm()` function.
 
 To cancel a specific scheduled alarm, use the `alarm_cancel()` function with the alarm ID. To cancel all alarms registered by the application, use the `alarm_cancel_all()` function.
 
+To update a specific scheduled alarm, use the alarm_update_delay(), alarm_update_date(), alarm_update_period(), and alarm_update_week_flag() with the alarm ID and proper update value.
```

Comment:
Kindly write functions within code tag like ‘alarm_update_delay()’. Please check the screenshot attached for the same:
![image](https://user-images.githubusercontent.com/42992085/61711384-8a04f080-ad71-11e9-977d-4d61495f45d9.png)

Please rephrase to:
To update a specific scheduled alarm, use `alarm_update_delay()`, `alarm_update_date()`, `alarm_update_period()`, and `alarm_update_week_flag()` with the alarm ID and proper update value.

Reply:
Done